coluutils.js

```diff
@@ -1514,6 +1514,11 @@ coluutils.requestParseTx = function requestParseTx(txid, headersToForward)
       return btc.mul(100000000);
     }
 
+    function getHeadersToForward() {
+      var context = session.get('context')
+      return context && context.req && context.req.service && context.req.service.headersToForward
+    }
+
```

Comment:
Every function before `client.methods...` is called (node-rest-client call): it expands the request headers.
For example: see line 1406.